announcements/1.16.0.md

```diff
@@ -0,0 +1,100 @@
+
+### WebRTC streaming improvements
+
+To further allow better fine-tuning of the WebRTC based streaming implementation in Anbox Cloud, 1.16 includes various small improvements:
+
+* Video bitrate limits can now be specified on a per resolution/frame rate level.
+* Additional options allow enabling NVIDIA-specific options for H.264 encoding ([adaptive quantisation](https://docs.nvidia.com/video-technologies/video-codec-sdk/nvenc-video-encoder-api-prog-guide/#adaptive-quantization-aq), [multipass](https://docs.nvidia.com/video-technologies/video-codec-sdk/nvenc-video-encoder-api-prog-guide/#multi-pass-frame-phencoding)).
+* Audio and video are now sent on separate streams to improve playout delay on the client.
+* The benchmark utility can now provide statistics for minimum playout and jitter buffer delay.
+* TURN credentials for the container are now dynamically allocated and are no longer created with a constant lifetime.
+
+### Data channel v2
+
+[Version 2 of the OOB data exchange protocol](https://anbox-cloud.io/docs/howto/stream/oob-data#oob-v2), which Anbox Cloud introduced in version 1.15, provides a full-duplex bidirectional data transmission between the Android container and the WebRTC client. In the 1.16 release, Anbox Cloud introduced the Anbox WebRTC data proxy service as an extension to version 2 of the OOB data. The data proxy service exposes a binder service in the Android container that can be used by an Android application to communicate with a WebRTC client over the Anbox WebRTC platform.
+
+### Ubuntu 22.04 support
+
+The Juju charms of Anbox Cloud now fully support Ubuntu 22.04. The Anbox Cloud Appliance can run on Ubuntu 22.04 already since the 1.15 release.
+
+By default, the Anbox Cloud bundles ([`anbox-cloud`](https://charmhub.io/anbox-cloud), [`anbox-cloud-core`](https://charmhub.io/anbox-cloud-core)) available from [Charmhub](https://charmhub.io/) use Ubuntu 22.04.
+
+With the introduction of Ubuntu 22.04, support for Ubuntu 18.04 was removed.
+
+### AMS improvements
+
+#### APT mirror configuration
+
+The APT mirror used inside Anbox containers has been hard-coded to `archive.ubuntu.com` or `ports.ubuntu.com`, which is not always ideal. To allow for easier modification, AMS now provides a `container.apt_mirror` configuration option. This option can be set to a custom Ubuntu mirror at runtime and will then be used by all containers that are started afterwards.
+
+#### Networking ACLs
+
+A new AMS charm configuration `use_network_acl` has been added. This configuration enables AMS to use [LXD network ACLs](https://linuxcontainers.org/lxd/docs/master/howto/network_acls/) <!-- wokeignore:rule=master --> rather than the AMS node controller (which is used by default) to restrict the network access of Anbox containers. The option is experimental and not ready for general use, but will in the future replace the functionality that is currently part of the AMS node controller.
```

Comment:
```suggestion
A new AMS charm configuration `use_network_acl` has been added. This configuration enables AMS to use [LXD network ACLs](https://linuxcontainers.org/lxd/docs/latest/howto/network_acls/) rather than the AMS node controller (which is used by default) to restrict the network access of Anbox containers. The option is experimental and not ready for general use, but will in the future replace the functionality that is currently part of the AMS node controller.
```